test/linalg/matmul.jl

```diff
@@ -5,189 +5,279 @@ using Base.Test
 ## Test Julia fallbacks to BLAS routines
 
 # matrices with zero dimensions
-for i = 1:10
```

Comment:
Okay. I'm not quite sure if I understand why you would have to loop ten times in that case either, but I'll look at it again.